### ISSUE TITLE
Websockets SSL / Server CPanel  

### ISSUE THREAD

**Issue by jakarman** [emqx/emqx#3756]
**Environment**
OS: CentOs 7.8
EMQ: EMQ: 4.2.0

**Description**

I'm trying to configure emqx on a VPS server with CPanel / whm (I have root access) but when trying to give SSL functionality to Websockets the connection doesn't work.

**Steps:**

0.- I opened ports 18083, 8084, 8083, 1883, 8883.
1.- I installed emqx with the commands.
     `sudo yum install -y yum-utils device-mapper-persistent-data lvm2`
      `sudo yum-config-manager --add-repo https://repos.emqx.io/emqx-ce/redhat/centos/7/emqx-ce.repo`
       `sudo yum install emqx`

2.- I went to `etc/emqx/certs` and renamed the _cert.pem_ to _cert_old.pem_ and _key.pem_ to _key_old.pem_

3.- I made a symbolic copy of the SSL of my Cpanel account (the SSL are provided by cPanel (powered by Sectigo) and managed by AutoSSL). using the commands
```
ln -s /home/<my_cpanel_acount>/ssl/certs/<my_crt>.crt /etc/emqx/certs/cert.pem
ln -s /home/<my_cpanel_acount>/ssl/keys/<my_key>.key /etc/emqx/certs/key.pem
```

4. The cert and key certificates appear in light blue at the `/etc/emqx/certs/`.

5. I initialized emqx using `emqx start` and get a correct log.

```
Starting emqx on node emqx@127.0.0.1
Start http:management listener on 8081 successfully.
Start http:dashboard listener on 18083 successfully.
Start mqtt:tcp listener on 127.0.0.1:11883 successfully.
Start mqtt:tcp listener on 0.0.0.0:1883 successfully.
Start mqtt:ws listener on 0.0.0.0:8083 successfully.
Start mqtt:ssl listener on 0.0.0.0:8883 successfully.
Start mqtt:wss listener on 0.0.0.0:8084 successfully.
EMQ X Broker 4.2.0 is running now!
```

6. I went to the dashboard <my_domain>: 18083 and in the Tools> Websockets section I make a test connection without using SSL (port 8083) The connection is correct.

7.- I disconnected the active connection and select the SSL check, which changes the port to 8084, I press the connect button and the following error appears in the browser console. I check the logs in `var/log/emqx` and there are no new logs.
![errorsww](https://user-images.githubusercontent.com/25474611/94724915-99da7600-0320-11eb-9cda-1326b3c0b2e5.png)

Could you help me, please? :)

**Comment by HJianBo** [comment]
Hi @jakarman Can you try to capture the 8084 network traffic and paste it here?

**Comment by jakarman** [comment]
Hi @HJianBo , 

An apology, I had forgotten to update this before.

I did exactly that and found that the communication was rejecting it due to the lack of a valid key file.

After that, I realized that the file I was copying did not have read permissions for other users ... so it was enough to give the file permissions and like magic, everything was fixed.

at least I hope this post works as a tutorial on how to install emqx in Cpanel.
